mainapp/tests.py

```diff
@@ -209,3 +214,36 @@ def test_user_rated_book_valid(self):
         self.assertEquals(rating.bookrating, valid_bookrating)
         self.assertEquals(rating.user, self.user)
         self.client.logout()
+
+class MostCommonGenreTestCase(TestCase):
```

Comment:
We can review this after we are on the same page with the new logic for the recommendations function

Reply:
Regarding the test don't hardcode the book ids, just take the first 5, then the next 5 and so on as we discussed in discord from the dataframe.
Btw do ping us if you need help

Reply:
Make a wrapper for all the tests which takes in parameters to avoid duplicate code